src/Views/Components/AdminUI/FormElements/index.tsx

```diff
@@ -17,50 +21,60 @@ export type FormElementProps = {
     id: string;
 };
 
-export const Form: React.FC<HTMLFormElement | FormElementProps> = ({children, id, onSubmit}) => (
+const Form: React.FC<HTMLFormElement | FormElementProps> = ({children, id, onSubmit}) => (
     <form className={styles.form} id={id} onSubmit={onSubmit}>
         {children}
     </form>
 );
 
+export {Form};
+
 /**
  *
  * @unreleased
  */
 
-export type InputFieldProps = {
+export type TextInputFieldProps = {
     name: string;
     type: string;
     placeholder: string;
     label: string;
 };
 
-export const TextInputField = React.forwardRef<HTMLInputElement, InputFieldProps>(
+const TextInputField = React.forwardRef<HTMLInputElement, TextInputFieldProps>(
     ({name, type, placeholder, label, ...props}, ref) => {
         return (
-            <FieldLabel label={label}>
+            <label>
+                {label && <span className={styles.fieldLabel}>{label}</span>}
                 <div className={cx(styles.textFieldContainer)}>
                     <input ref={ref} name={name} type={type} placeholder={placeholder} {...props} />
                 </div>
-            </FieldLabel>
+            </label>
         );
     }
 );
 
+export {TextInputField};
+
 /**
  *
  * @unreleased
  */
 
-export type CurrencyInputFieldProps = InputFieldProps & {
+export type CurrencyInputFieldProps = {
```

Comment:
I think we might have overlooked something in the merge. This is no longer extending the InputfieldProps but instead reverting back tot he original way is was setup. Should I correct it here in this PR or readdress it later?

Reply:
Resolved e7ea4ed030da00d818b841bd6e86ff564fcb24eb

Reply:
Great that makes sense. Nice job.